go.mod

```diff
@@ -11,6 +11,6 @@ require (
 require golang.org/x/text v0.14.0 // indirect
 
 retract (
-    v0.1.1 // contains invalid module name 
-    v0.1.2 // contains only retractions
+	v0.1.2 // contains only retractions
```

Comment:
`go mod tidy` check in GitHub Actions [failed](https://github.com/franchb/gobergamot/actions/runs/8036734755/job/21950791775#step:8:54), so I've run `go mod tidy` in the `main` branch of `github.com/KSpaceer/gobergamot` and pushed the updated go.mod and go.sum files.